spec/system/candidate_interface/candidate_replaces_reference_after_applying_again_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate applying again' do
+  include CandidateHelper
+
+  around do |example|
+    date_that_avoids_clocks_changing_by_ten_days = Time.zone.local(2020, 1, 13)
+    Timecop.freeze(date_that_avoids_clocks_changing_by_ten_days) do
+      example.run
+    end
+  end
```

Comment:
Do we need this bit?

Reply:
Good point, that's not needed in this spec because there are not date based assertions.